### ISSUE TITLE
Editor introduces `text` property in `layout.shape` which is not valid in plotly.js schema

### ISSUE THREAD

**Issue by jonmmease** [plotly/react-chart-editor#748]
To reproduce:

1. Open this figure in the editor

```
{"data":[{"type":"scatter","y":[2,4,3,2.5]}],"layout":{}}
```

2. Add a shape and then examine figure state
```
{"data":[{"type":"scatter","y":[2,4,3,2.5],"mode":"lines+markers"}],

 "layout":{...
    "shapes":[{"text":"Shape 0","line":{"color":"#444444"},"fillcolor":"#7f7f7f","opacity":0.3}]
  },

 "frames":[]}
```

The shape that is created has a `text` attribute, but this is not part of the plot schema and causes a validation error when loaded by plotly.py

**Comment by nicolaskruchten** [comment]
Same with image panel :)

**Comment by nicolaskruchten** [comment]
Closed by #749
